docs/include.am

```diff
@@ -7,85 +7,89 @@
 
 SPHINXOPTS    = ${SPHINX_WARNINGS} -q
 PAPER         =
-SPHINX_SOURCEDIR = ${abs_srcdir}/docs
-SPHINX_BUILDDIR  = ${abs_srcdir}/sphinx-build
+SPHINX_SOURCEDIR = ${abs_top_srcdir}/docs
+SPHINX_BUILDDIR = ${abs_builddir}/docs
```

Comment:
> Is this supposed to be docs or sphinx-build?

It should be renamed to `docs`. Before the output directory was a toplevel directory called `sphinx-build`. However it was a bit confusing since the binary is also called `sphinx-build`